frontend/src/v5/ui/components/viewer/cards/cardFilters/cardFilters.types.ts

```diff
@@ -16,4 +16,14 @@
  */
 
 export type CardFilterOperator = 'ex' | 'nex' | 'eq' | 'neq' | 'ss' | 'nss' | 'rng' | 'nrng' | 'gt' | 'gte' | 'lt' | 'lte';
-export type CardFilters = Record<string, any[]>;
\ No newline at end of file
+export type CardFilterType = 'text' | 'longText' | 'date' | 'pastDate' | 'sequencing' | 'oneOf' | 'manyOf' | 'boolean' | 'number' | 'ticketTitle' | 'ticketId' | 'template';
+export type CardFilterValue = string | number | Date;
+export type BaseFilter = { operator: CardFilterOperator, values: CardFilterValue[] };
+
+export type CardFilter = {
+	property: string,
+	type: CardFilterType,
+	filter: BaseFilter,
+};
+export type TicketCardFilter = CardFilter & { module: string };
+export type TicketCardFilterDescription = Omit<TicketCardFilter, 'filter'>;
```

Comment:
I dont think is necessary to have 3 types here. Just with one you can cater for all the cases:

```
export type CardFilter = {
	property: string,
	type: CardFilterType,
	filter?: BaseFilter,
	module?: string,
};
```